templates/kubernetes/index.html

```diff
@@ -35,6 +35,27 @@ <h3>Fault Tolerant</h3>
   </div>
 </section>
 
+<section class="p-strip is-bordered">
+  <div class="row p-divider">
+    <div class="col-6 p-divider__block">
+      <h2>In partnership with Google&nbsp;GKE</h2>
+      <p>Google and Canonical together enable smooth hybrid operations between Google&rsquo;s Container Engine (GKE) service with Ubuntu worker nodes, and Canonical&rsquo;s Distribution of Kubernetes. Choose Canonical&rsquo;s Kubernetes to be sure your container workloads can migrate to GKE thanks to our kernel-to-k8s&nbsp;alignment.</p>
+      <p><a href="https://cloud.google.com/container-engine/docs/node-images#ubuntu_beta" class="p-link--external">Try GKE with Ubuntu now</a></p>
```

Comment:
This should be https://cloud.google.com/kubernetes-engine/docs/node-images#ubuntu. Ubuntu is now out of beta.

Reply:
Thank you, done and updated doc